database/CMakeLists.txt

```diff
@@ -182,9 +182,10 @@ else()
   set(additional_link_LIB
       flatbuffers
       firebase_rest_lib
-      ${OPENSSL_CRYPTO_LIBRARY}
       leveldb
-      libuWS)
+      libuWS
+      OpenSSL::SSL
```

Comment:
Same as before. Do we need to state this dependency explicitly?

Reply:
Without this, the tests were failing to link on Linux.